### ISSUE TITLE
Status definition 

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#22]
## User story
1. As a user (HR employee)
2. I want / need to know the progress of registration
3. So that I can understand current status of the ongoing or finalized registrations. 

## Acceptance criteria
* Status 1: HR employee started account for new employee ⇨ "account initialized"
* Status 2: e-mail is sent to employee ⇨ "waiting for information from new employee"
* Status 3: Digital identity is created ⇨ "Registered"

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
